Dangerfile.swift

```diff
@@ -0,0 +1,144 @@
+}
+
+// Warn, asking to update Chinese docs if only English docs are updated and vice-versa
+let enDocsModified = danger.git.modifiedFiles.first { $0.contains("docs/") } != nil
+let cnDocsModified = danger.git.modifiedFiles.first { $0.contains("docs_CN/") } != nil
+if (enDocsModified && !cnDocsModified) || (!enDocsModified && cnDocsModified) {
+    warn("Consider **also** updating the \(enDocsModified ? "English" : "Chinese") docs. For Chinese translations, request the modified file(s) to be added to the list [here](https://github.com/Moya/Moya/issues/1357) for someone else to translate, if you can't do so yourself.")
+}
+
+// Warn, asking to update Chinese README if only English README are updated and vice-versa
+let enReameModified = danger.git.modifiedFiles.first { $0.contains("Readme.md") } != nil
+let chReameModified = danger.git.modifiedFiles.first { $0.contains("Readme_CN.md") } != nil
+if (enReameModified && !chReameModified) || (!enReameModified && chReameModified) {
+    warn("Consider **also** updating the \(enReameModified ? "Chinese" : "English") README. For Chinese translations, request the modified file(s) to be added to the list [here](https://github.com/Moya/Moya/issues/1357) for someone else to translate, if you can't do so yourself.")
+}
+
+// Warn when there is a big PR
+if (danger.github.pullRequest.additions ?? 0) > 500 {
+    warn("Big PR, try to keep changes smaller if you can")
+}
+
+// Don't let testing shortcuts get into master by accident
+if danger.utils.exec("grep -r \"fit Demo/Tests/\"").count > 1 {
+    fail("fit left in tests")
+}
+
+// Added (or removed) library files need to be added (or removed) from the
+// Carthage Xcode project to avoid breaking things for our Carthage users.
+let addedSwiftLibraryFiles = danger.git.createdFiles.first { $0.fileType == .swift && $0.hasPrefix("Sources") } != nil
+let deletedSwiftLibraryFiles = danger.git.deletedFiles.first { $0.fileType == .swift && $0.hasPrefix("Sources") } != nil
+let modifiedCarthageXcodeProject = danger.git.modifiedFiles.first { $0.contains("Moya.xcodeproj") } != nil
+if (addedSwiftLibraryFiles || deletedSwiftLibraryFiles) && !modifiedCarthageXcodeProject {
+    fail("Added or removed library files require the Carthage Xcode project to be updated. See the Readme")
+}
+
+let missingDocChanges = danger.git.modifiedFiles.first { $0.contains("docs") } != nil
+let docChangeRaccomanded = (danger.github.pullRequest.additions ?? 0) > 15
+if sourceChanges && missingDocChanges && docChangeRaccomanded && !isTrivial {
+    warn("Consider adding supporting documentation to this change. Documentation can be found in the `docs` directory.")
+}
+
+// Run danger-prose to lint Chinese docs
+func isCnDocsMarkdown(_ file: String) -> Bool {
+    return file.fileType == .markdown && file.contains("docs_CN")
+}
+let addedAndModifiedCnDocsMarkdown = danger.git.createdFiles.filter(isCnDocsMarkdown) + danger.git.modifiedFiles.filter(isCnDocsMarkdown)
+if #available(OSX 10.12, *),
```

Comment:
Because `Proselint` is using https://developer.apple.com/documentation/foundation/filemanager/1642807-homedirectoryforcurrentuser
(Proselint wants the config in the user home :( https://github.com/amperser/proselint#checks)